chain-extensions/types/xvm/src/lib.rs

```diff
@@ -18,7 +18,7 @@
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-use astar_primitives::xvm::CallError;
+use astar_primitives::{xvm::CallError, Balance};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

It seems that the import statement has changed. The import of `astar_primitives::xvm::CallError` has been replaced with `astar_primitives::{xvm::CallError, Balance};`. This addition includes the `Balance` type as well. Ensure that this change does not break any other parts of the codebase that rely on `CallError`. Also, make sure that the import of `Balance` is indeed required for the following code changes.
```diff
+use astar_primitives::{xvm::CallError, Balance};
 use parity_scale_codec::{Decode, Encode};
 use sp_std::vec::Vec;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->